src/Module.ts

```diff
@@ -753,4 +753,13 @@ export default class Module {
 		warning.id = this.id;
 		this.graph.warn(warning);
 	}
+
+	shimMissingExport(name: string) {
+		// could have already been generated
+		if (!this.exports[name])
+			this.exports[name] = {
+				localName: '_shimmedExport'
+			};
+		return this.graph.scope.findVariable('_shimmedExport');
```

Comment:
Instead of searching we might just use `this.graph.exportShimVariable` here which would also be a little clearer.